src/EditorFeatures/Core.Wpf/Preview/PreviewFactoryService.cs

```diff
@@ -374,7 +382,9 @@ private async ValueTask<DifferenceViewerPreview> CreateRemovedDocumentPreviewVie
             // the diff view would long have been dismissed by the time user edits the code)
             // resulting in crashes. Instead we create a new buffer from the same content.
             // TODO: We could use ITextBufferCloneService instead here to clone the original buffer.
+#pragma warning disable CA2007 // Consider calling ConfigureAwait on the awaited task (containing method uses JTF)
```

Comment:
Do you want to just pragma the entire file?

Reply:
I'm OK with this approach for now. We need to find a better long-term solution though.